utils/python/tests/scripts_regression_tests

```diff
@@ -23,7 +23,7 @@ NO_BATCH    = False
 logging.basicConfig()
 
 ###############################################################################
-class RunUnitTests(unittest.TestCase):
+class A_RunUnitTests(unittest.TestCase):
```

Comment:
Tests are run in sort order of test names - I've added some letters to the beginning of classes to control order a little.  There is no dependence between tests but faster smaller ones are run first - if they fail you can probably abort and fix them before going on to the larger slower ones.